odoo/addons/base/views/ir_qweb_widget_templates.xml

```diff
@@ -25,7 +25,7 @@
         <div class="gap-2" itemprop="address" itemscope="itemscope" itemtype="http://schema.org/PostalAddress">
             <div t-if="address and 'address' in fields" t-attf-class="d-flex align-items-baseline gap-1">
                 <i t-if="not options.get('no_marker')" class="fa fa-map-marker fa-fw" role="img" aria-label="Address" title="Address"/>
-                <span class="d-block w-100 lh-sm" itemprop="streetAddress" t-esc="address"/>
+                <span class="w-100 lh-sm text-truncate" itemprop="streetAddress" t-esc="address"/>
```

Comment:
Yes, it absolutely makes sense to display the address in multiple lines rather than truncate it. However, if the address is too long, displaying the entire address could be quite messy. What do you think? The specifications mentioned using ellipsis, so maybe we should confirm with Lna to see what he thinks? :)